dashboard/views.py

```diff
@@ -52,11 +55,10 @@ def get(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-s
         # create an instance of the client to query edX
         edx_client = EdxApi(user_social.extra_data, settings.EDXORG_BASE_URL)
         # get an enrollments client for the student
-        enrollments = edx_client.enrollments.get_student_enrollments()
+        course_keys = CourseRun.objects.filter(course__program__live=True).values_list("edx_course_key", flat=True)
```

Comment:
you need to filter by `edx_course_key` not null or not empty:

I found a bug in my local instance where 
```
course_keys == ['course-v1:odl+MX+2015', 'course-v1:edX+DemoX+Demo_Course', 'course-v1:odl+GIO101+CR-FALL15', 'course-v1:odl+GIO101+FALL13', 'course-v1:odl+GIO101+FALL14', '']
```
note the empty string at the end of the list